src/Raven.Studio/typescript/commands/database/tasks/getOpenAiModelsCommand.ts

```diff
@@ -0,0 +1,49 @@
+import commandBase = require("commands/commandBase");
+import yup = require("yup");
+
+class getOpenAiModelsCommand extends commandBase {
+    private readonly defaultEndpoint = "https://api.openai.com/v1/";
```

Comment:
Why are you making that call from the client? That should be from the server side, no?

Also - what happens when I use Grok / Mystral / etc?

Reply:
User can provide URL in connection string and it will be used as an endpoint here

Reply:
This isn't limited to just the OAI endpoint?

Note that this will fail in cases where the client cannot access the endpoint (and vice versa - client can, server can't).

Imagine that we have Ollama running locally, the endpoint is `localhost`, but we are accessing the server from another machine.
That is a problem, no?

Reply:
But why? 
`http://localhost:11434/v1/models` is perfectly valid - and we _want_ that.